src/ChipInput.js

```diff
@@ -343,6 +343,16 @@ class ChipInput extends React.Component {
         }
       }
 
+      if (typeof chip === 'object') {
+        if (chip[this.props.dataSourceConfig.text].length === 0 || chip[this.props.dataSourceConfig.value].length === 0) {
```

Comment:
This can be removed, it's already handled below.

Reply:
You mean line 347? Or both?

Reply:
Removing everything would revert your PR. :smile: I mean lines 347–349

Reply:
It's not handled, because if we will not add such checking - we will get an empty chip by pressing "enter"
![dec-28-2017 17-39-53](https://user-images.githubusercontent.com/7280838/34415230-2c212364-ebf6-11e7-8f50-24dabdac86ba.gif)

Reply:
Oh, you're right. It's fine then.